src/latexify/codegen/function_codegen.py

```diff
@@ -356,6 +356,17 @@ def visit_Call(self, node: ast.Call) -> str:
             (default_func_str + r"\mathopen{}\left(", r"\mathclose{}\right)"),
         )
 
+        if func_name == "zeros":
```

Comment:
- For `zeros`, We don't need to constrain only a particular number of dimensions.
- I think these processes don't work if the subtree has unexpected syntax. It usually happens when users gave other functions with the same name. As the AST varies, we basically need complete check of the underlying structure of the given subtree.